src/utils/OWLConversor/OWLConversor.js

```diff
@@ -0,0 +1,145 @@
+import './xml2json.js';
+import './Utils.js';
+import './Query.js';
+
+(function(){
+	window.MAX_OBJECTS_PER_ROW = 5;
+	
+	var x2js = new X2JS({attributePrefix :"@"});
+	
+	var Config = require("./config.terms.json");
+	var IgnoreList = null;
+	var RemoveList = null;
+	
+	var fn = {
+		convert : function(ontology){
+			Utils.reset();
+			Query.reset();
+			Query.setData(ontology);
+			
+			var convertedData = fn.getDefaultProperties();
+			
+			convertedData.network.name = Query("owlName");
+			convertedData.nodes = Query("nodes");
+			
+			Utils.setConvertedData(convertedData);
+			convertedData = Utils.applyDisjoints();
+			
+			// somente adiciona o cpt após remover os elements "desnecessários" para não gerar cpt a mais
+			for(var x = 0; x < convertedData.nodes.length; x++){
```

Comment:
[Check this site](https://hacks.mozilla.org/2015/04/es6-in-depth-iterators-and-the-for-of-loop/).

```js
fot (const node of convertedData.nodes) {
...
}
```